test_runner/batch_others/test_branching.py

```diff
@@ -38,20 +38,16 @@ def test_branching_with_pgbench(neon_simple_env: NeonEnv,
              'pitr_interval': '5 s'
          })
 
-    def run_pgbench(pg: Postgres):
-        connstr = pg.connstr()
-
-        log.info(f"Start a pgbench workload on pg {connstr}")
-
-        pg_bin.run_capture(['pgbench', '-i', f'-s{scale}', connstr])
```

Comment:
When using `pg_current_wal_flush_lsn` to create the branch, I encountered an error in this test that resulted from a statement timeout when trying to drop tables for `pgbench` initialization. I suspect the error was because of some race condition during `pgbench` initialization on different branches. This change is to sequentialize the initialization step for branches.

Reply:
Hmm, it would be good to dig deeper into that. Do you suspect it was a race condition in the pageserver? That sounds like a bug we should find and fix. I don't see anything inherently wrong with this test as it was.

Or maybe running multiple pgbench initializations just overloads the server, so that each initialization takes a long time? In that case, a longer `statement_timeout` would help. Or making things faster if possible. The slowest SQL command that 'pgbench -i` runs is probably a CREATE INDEX. (Or ALTER TABLE ADD PRIMARY KEY, which is essentially the same thing). That can legitimately take a long time, proportional to the scale factor. How long does it take in this test? How close is it to statement_timeout?